find-phantom-resource.ps1

```
@@ -129,4 +434,156 @@ az resource list --query "[?managedBy!='']" -o table
+
+            Write-Host "`n  Deleted: $deletedCount, Failed: $failedCount" -ForegroundColor $(if ($failedCount -gt 0) { "Yellow" } else { "Green" })
+
+            if ($DeleteResourceGroup -and $failedCount -eq 0) {
+                # Re-check if the RG is now empty
+                $remainParsed = Invoke-AzRest -Url "/subscriptions/$subscriptionId/resourceGroups/$ResourceGroup/resources?api-version=2024-03-01"
+                $remaining = 0
+                if ($null -ne $remainParsed) {
+                    $remaining = $remainParsed.value.Count
+                }
+                if ($remaining -eq 0) {
+                    Write-Host "`n  Resource group is empty. Deleting $ResourceGroup..." -NoNewline
```

PSScriptAnalyzer (1.24.0)

[warning] File 'find-phantom-resource.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)